core/src/main/scala/org/broadinstitute/dsde/rawls/config/MultiCloudWorkspaceConfig.scala

```diff
@@ -34,12 +38,15 @@ case object MultiCloudWorkspaceConfig {
     conf.getConfigOption("multiCloudWorkspaces") match {
       case Some(mc) =>
         new MultiCloudWorkspaceConfig(
-          conf.getBoolean("multiCloudWorkspaces.enabled"),
-          util.toScalaDuration(conf.getDuration("multiCloudWorkspaces.cloudContextPollTimeoutSeconds")),
+          mc.getBoolean("enabled"),
+          Some(MultiCloudWorkspaceManagerConfig(
+            mc.getString("workspaceManager.leonardoWsmApplicationId"),
```

Comment:
Where does `leonardoWsmApplicationId` get defined? I'm not finding it in any repo.

Reply:
I mean, I understand that's a configuration string value. But how do we know this is the key we should be using?

Reply:
Why wouldn't it be an "id" under Leo in that block? I'm still not clear how it will map to the configuration.

Reply:
So WSM configures this [here](https://github.com/DataBiosphere/terra-workspace-manager/blob/8faa5f96debd9144eb38837a1bf1f19d3e1650a0/service/src/main/java/bio/terra/workspace/service/workspace/WsmApplicationService.java#L266). It's using the top level identifier ("leo" in this case) as the applicationId

Reply:
This still seems like magic to me, but I'll take your word for it.